certified-connectors/Airmeet/apiDefinition.swagger.json

```diff
@@ -850,216 +869,6 @@
-              "value-collection": "data",
-              "value-title": "name",
-              "parameters": {
-                "allAirmeets": true,
-                "crmName": "MICROSOFT_DYNAMICS"
-              }
-            }
-          },
-          {
-            "name": "body",
-            "in": "body",
-            "required": false,
-            "schema": {
-              "type": "object",
-              "properties": {
-                "sessionTitle": {
-                  "type": "string",
-                  "description": "Title for the Session",
-                  "title": "Session Title"
-                },
-                "sessionStartTime": {
-                  "type": "integer",
-                  "format": "int64",
-                  "description": "Start time for the session, unix timestamp rounded to minutes",
-                  "title": "Session Start Time"
-                },
-                "sessionDuration": {
-                  "type": "integer",
-                  "format": "int32",
-                  "description": "Duration of the event in seconds",
-                  "title": "Session Duration"
-                },
-                "sessionSummary": {
-                  "type": "string",
-                  "description": "Summary for the session",
-                  "title": "Session Summary"
-                },
-                "hostEmail": {
-                  "type": "string",
-                  "description": "Email of the Airmeet event host",
-                  "title": "Host Email"
-                },
-                "speakerEmails": {
-                  "type": "array",
-                  "x-ms-visibility": "advanced",
-                  "items": {
-                    "type": "string"
-                  },
-                  "description": "Emails of all the speakers. Speakers should be added to the Airmeet before creating the session"
-                },
-                "cohostEmails": {
-                  "type": "array",
-                  "x-ms-visibility": "advanced",
-                  "items": {
-                    "type": "string"
-                  },
-                  "description": "Emails of all the co hosts. Team members should be added to the community before creating the session"
-                },
-                "type": {
-                  "type": "string",
-                  "description": "type",
-                  "title": "Type",
-                  "default": "HOSTING",
-                  "x-ms-visibility": "internal"
-                }
-              },
-              "required": [
-                "hostEmail",
-                "sessionTitle",
-                "sessionStartTime",
-                "type"
-              ]
-            }
-          }
-        ],
-        "summary": "Add a Session to your Airmeet",
-        "description": "Create Session",
-        "operationId": "CreateSession"
-      }
-    },
-    "/airmeet/{airmeetId}/attendee": {
```

Comment:
@amit-vb This operation has been deleted. This change breaks existing flows.
can you please confirm that no users will be affected by this change?

Reply:
Hi @Amjed-Ayoub 
right now we have only internal users & so after the new version will send them a notification to use the latest versions.